README.md

```diff
@@ -18,14 +18,16 @@ policy for the provided forms of distribution.
 
 ## Set up a node and/or baking on Ubuntu
 
+> :warning: Since [#570](https://github.com/serokell/tezos-packaging/pull/570), wizards' names were simplified.
+
```

Comment:
and with the two previous suggestions of course this is no longer necessary:
```suggestion
> :warning: Since [#570](https://github.com/serokell/tezos-packaging/pull/570), wizards' names were simplified.

```